### PR TITLE
Issue 3409 animation fill mode add media

### PR DESCRIPTION
**Pull request by DustyDogCodex**
### Description

Closes issue #3409 

Added gifs showcasing animation-fill-mode: forwards and animation-fill-mode: backwards to media folder.
Updated animation-fill-mode.md with these gifs.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by DustyDogCodex** [comment]
hi @KTom101 !

Hope you are doing well! I have added gifs showcasing the appropriate animations and updated the animation-fill-mode.md file with those gifs as well. Please let me know if I need to change anything here.

Thanks!

File: content/css/concepts/animations/terms/animation-fill-mode/animation-fill-mode.md
Comment (SSwiniarski) [inline content/css/concepts/animations/terms/animation-fill-mode/animation-fill-mode.md:1]: Looks good, but you need to update the links as specified in the [file hosting section of the style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting). This is so the images will show when the page is published to the Docs site.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DustyDogCodex I did a first review and only had one comment.

**Comment by DustyDogCodex** [comment]
Hi @SSwiniarski !

Hope you are having a good day of coding today! I have made your recommended changes to the gif links. They should now be in line with the style guide. Please review them when you have a moment and let me know if this PR needs anymore changes.

Thanks!

**Review by SSwiniarski — APPROVED** [review]
@DustyDogCodex Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@DustyDogCodex @SSwiniarski LGTM

**Pull request merged**